libs/design/guides/getting-started.md

```diff
@@ -9,12 +9,28 @@ This tutorial is designed to touch upon the very basics of the `@daffodil/design
 
 ### npm
 ```bash
-npm install @daffodil/design --save
+npm install --save @daffodil/design @daffodil/core \
```

Comment:
This needs qualification that its only applicable to node <= 12.

The old commands are valid for node >=14.

Reply:
updated